api/src/main/resources/messages_fr.properties

```diff
@@ -65,6 +65,7 @@ adminui.privilege.saved=Enregistrer privilège
 adminui.error.privilege.save.fail=Échec de l'enregistrement du privilège
 adminui.privilege.retired=Privilège inactif
 adminui.privilege.children=Privilège enfant 
+adminui.privilege.unknown=Aucun privilège trouvé avec le nom
```

Comment:
Hey @HerbertYiga, do you think we need this

Reply:
add that to messages. Properties file with the English message ,run your changes in a running server to ensure it works as required

Reply:
Year it is displaying as required.  
thank you @HerbertYiga